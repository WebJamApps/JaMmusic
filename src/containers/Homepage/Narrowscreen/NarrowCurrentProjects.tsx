import React from 'react';
import CollegeLutheran from '../CollegeLutheran';
import AppersonAutomotive from '../AppersonAutomotive';

const NarrowCurrentProjects = (): JSX.Element => (
  <div className="notWidescreen">
    <div className="material-content">
      <h3 style={{ textAlign: 'center', marginTop: '15px', marginBottom: '15px' }}>Our Current Projects</h3>
      <CollegeLutheran />
      <p style={{ fontSize: '6pt', marginBottom: '0' }}>&nbsp;</p>
      <AppersonAutomotive />
    </div>
  </div>
);

export default NarrowCurrentProjects;
