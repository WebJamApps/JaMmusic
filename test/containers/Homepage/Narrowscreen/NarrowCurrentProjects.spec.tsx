import React from 'react';
import { shallow } from 'enzyme';
import NarrowCurrentProjects from '../../../../src/containers/Homepage/Narrowscreen/NarrowCurrentProjects';

const wrapper = shallow<typeof NarrowCurrentProjects>(<NarrowCurrentProjects/>);

describe('NarrowCurrentProjects', () => {
  it('Renders the NarrowCurrentProjects', () => {
    expect(wrapper.find('div.notWidescreen').exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
