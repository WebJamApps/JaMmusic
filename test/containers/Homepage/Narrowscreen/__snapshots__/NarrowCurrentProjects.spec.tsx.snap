// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NarrowCurrentProjects Renders the NarrowCurrentProjects 1`] = `ShallowWrapper {}`;
